validator/client/runner.go

```diff
@@ -117,9 +117,6 @@ func run(ctx context.Context, v Validator) {
 			// Report this validator client's rewards and penalties throughout its lifecycle.
 			log := log.WithField("slot", slot)
 			log.WithField("deadline", deadline).Debug("Set deadline for proposals and attestations")
-			if err := v.LogValidatorGainsAndLosses(slotCtx, slot); err != nil {
-				log.WithError(err).Error("Could not report validator's rewards/penalties")
-			}
```

Comment:
Why move this?

Reply:
This is part of #7708 so perhaps is better to keep the discussion there?